cmd/lima-guestagent/daemon_linux.go

```diff
@@ -58,7 +60,43 @@ func daemonAction(cmd *cobra.Command, _ []string) error {
 		// without depending on `bpftrace` binary.
 		// The agent binary will need CAP_BPF file cap.
 		ticker := time.NewTicker(tick)
-		return ticker.C, ticker.Stop
+
+		logrus.Info("register for sighup notifications")
```

Comment:
Nit: probably sighup should be spelled in the upper chars, as it might look like "sigh up" (?) or a typo of sign-up